api/v1/serializers/developers-serializer.js

```diff
@@ -20,7 +20,7 @@ const developerResourceUrl = resourcePathLink(apiBaseUrl, developerResourcePath)
  * from the raw data rows.
  */
 _.forEach(developerResourceKeys, (key, index) => {
-  developerResourceKeys[index] = key.toUpperCase();
+  developerResourceKeys[index] = key;
```

Comment:
What does this `forEach` block do? Do you really need this block of code?

Reply:
With the aliases this is no longer needed so its removed.